lib/active_record/connection_adapters/spanner/database_statements.rb

```diff
@@ -35,7 +35,7 @@ def execute sql, name = nil, binds = []
               binds.delete staleness_hint
             end
 
-            log sql, name do
+            log sql, name, binds, type_casted_binds(binds) do
```

Comment:
I think we need to make this an opt-in in some way. Not logging the actual parameter values has been the behavior for quite some time, and suddenly changing it could be an unpleasant surprise to existing users, as the parameter values could contain sensitive data.

Reply:
This is how ActiveRecord works for every other database, but I can look into adding an opt-in option since it changes the behavior of this specific adapter.

I will also add some logging tests since there is an option.

Reply:
tests and option added